### PR TITLE
FPL-38: added enter applicant functionality to fpl ccd configuration

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPL-38

### Change description ###

Added fields to allow user to enter information about applicant and solicitor. Created page object to represent the form and written two e2e tests to fill in half the form, and the full form.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by swalker125** [comment]
screenshot_2018-10-09 core case data hmcts
screenshot_2018-10-09 core case data hmcts 1
screenshot_2018-10-09 core case data hmcts 2

File: e2e/config.js [hunk 1]
```diff
@@ -19,5 +19,5 @@ module.exports = {
 		attendingHearing: 'Attending hearing',
 		uploadDocuments: 'Annex documents',
 	},
-	testFile: './e2e/fixtures/mockFile'
-};
\ No newline at end of file
+	testFile: './e2e/fixtures/mockFile',
```
Comment (kurt-uk) [inline e2e/config.js]: trailing?

File: e2e/fixtures/solicitor.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	name: 'John Smith',
+	mobileNumber: '07000000000',
+	telephoneNumber: '00000000000',
+	email: 'solicitor@email.com',
+	dx: '160010 Kingsway 7 ',
+	reference: '#reference',
+};
```
Comment (kurt-uk) [inline e2e/fixtures/solicitor.js]: 1 fixture file with multiple objects?

File: e2e/pages/enterApplicant/enterApplicant.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+			name: '#enterApplicant_solicitorName',
+			mobileNumber: '#enterApplicant_solicitorMobile',
+			telephoneNumber: '#enterApplicant_solicitorTelephone',
+			email: '#enterApplicant_solicitorEmail',
+			dx: '#enterApplicant_solicitorDx',
+			reference: '#enterApplicant_solicitorReference',
+		}
+	},
+
+	enterApplicantDetails() {
```
Comment (geek-soft) [inline e2e/pages/enterApplicant/enterApplicant.js]: To make it more configurable I would suggest passing applicant in method args. This way here we get:

```
enterApplicantDetails(applicant) {
  ...
}
```

And in test file we have:

```
const applicant = require('../../fixtures/applicant.js');
```

Comment (swalker125) [reply e2e/pages/enterApplicant/enterApplicant.js]: Good spot 👍

File: e2e/paths/enterApplicant_test.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const config = require('../config.js');
+
+Feature('EnterApplicant');
```
Comment (geek-soft) [inline e2e/paths/enterApplicant_test.js]: Using plain English perhaps would be better .e.g. 'Enter applicant'.

**Pull request merged**